src/com/palmergames/bukkit/towny/invites/Invite.java

```diff
@@ -20,4 +22,13 @@
 	 */
 	TownyInviteSender getSender();
 
+	/**
+	 * @throws TownyException
```

Comment:
Needs  a description for why it throws, so that Javadocs doesn't throw an error when GitHub Actions attempts to build them. This can just be something generic.

Reply:
I've added a reason for the throws in the latest commit